docs/6.contributing.md

```diff
@@ -0,0 +1,30 @@
+### Contributing
```

Comment:
```suggestion
## Contributing
```
I think this will give us a heading "Contributing" in the left sidebar, and then the "Contributors" part will be a subsection of that. (At the moment, there are two links in the sidebar that link to the same page.)